superset-frontend/src/explore/components/SaveModal.tsx

```diff
@@ -75,25 +88,25 @@ class SaveModal extends React.Component {
     });
   }
 
-  onSliceNameChange(event) {
-    this.setState({ newSliceName: event.target.value });
+  onSliceNameChange(event: React.FormEvent<FormControl>) {
+    this.setState({ newSliceName: (event.target as HTMLInputElement).value });
```

Comment:
Maybe if the event is typed as `HTMLInputEvent` in the function args then this won't need to be cast here?